### PR TITLE
Add function to get admin user uuid

### PR DESCRIPTION
**Pull request by bhagyasakalanka**
Related Issues https://github.com/wso2/product-is/issues/12071

### CODE REVIEW

File: components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java [hunk 1]
```diff
@@ -379,10 +382,36 @@ public static void deleteTenant(String tenantDomain) throws Exception {
...
+            return adminUserUuid;
+        }
+        // If the admin user uuid is not in the tenant object, we need to get it from user store.
+        String adminUsername = tenant.getAdminName();
+        try {
+            UserStoreManager userStoreManager = (UserStoreManager) TenantMgtServiceComponent.getRealmService().
+                    getTenantUserRealm(tenant.getId()).getUserStoreManager();
+            adminUserUuid = ((AbstractUserStoreManager) userStoreManager).getUserIDFromUserName(adminUsername);
+        } catch (org.wso2.carbon.user.api.UserStoreException e) {
+            log.error(
```
Comment (somindatommy) [inline components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java:402]: Lets throw an server exception here

File: components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java [hunk 2]
```diff
@@ -379,10 +382,36 @@ public static void deleteTenant(String tenantDomain) throws Exception {
...
+                    String.format("Error occurred while getting user store manager for tenantId: %s", tenant.getId())
+            );
+        }
+        if (StringUtils.isBlank(adminUserUuid)) {
+            throw new UserStoreException(
+                    String.format("Error occurred while getting admin user unique identifier " +
```
Comment (somindatommy) [inline components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java:408]: Shall we say `No UUID found for the admin user of tenant: %s`. Also lets throw an server exception here

File: components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java (hunk 1 above)
Comment (bhagyasakalanka) [reply components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java]: fixed

File: components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java (hunk 2 above)
Comment (bhagyasakalanka) [reply components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java]: fixed

**Review by somindatommy — APPROVED** [review]

**Pull request merged**